Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -136,6 +136,68 @@ protected override void ProcessRecord()
 
     /// <summary>
     /// <para type="synopsis">
+    /// Updates information in an existing dataset.
+    /// </para>
+    /// <para type="description">
+    /// Updates information in an existing dataset. If no Project is specified, the default project will be used. 
+    /// This cmdlet returns a Dataset object.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> $updatedSet | Set-GcbqDataset</code>
+    ///   <para>This will update the values stored in the cloud for the dataset passed via pipeline.</para>
```

Comment:
Got it.  Added in both of those two lines, as well as updating the default table expiration time in the second example for added wonder and amazement.  Updated a lot of the wording as well to be more explicit.